pkgs/misc/meson64-tools/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, fetchFromGitHub, buildPackages }:
+
+stdenv.mkDerivation rec {
+  pname = "meson64-tools";
+  version = "unstable-2020-08-03";
+
+  src = fetchFromGitHub {
+    owner = "angerman";
+    repo = pname;
+    rev = "a2d57d11fd8b4242b903c10dca9d25f7f99d8ff0";
+    sha256 = "1487cr7sv34yry8f0chaj6s2g3736dzq0aqw239ahdy30yg7hb2v";
+  };
+
+  buildInputs = with buildPackages; [ openssl bison yacc flex bc python3 python2 ];
```

Comment:
Please don't mix python2 and python3.

Reply:
@SuperSandro2000, I did simple `s/python2/python/g` here. python2 roots from mbedtls dependency, and there are no bits of python that wouldn't work in python3. tested (tm).

Reply:
But why do we need python2 then?

Reply:
@SuperSandro2000, we don't, a script that runs tests in meson64-tools wants a patch. I will suggest it to the relevant repo later.